### ISSUE TITLE
Code for Point-to-Voxel Knowledge Distillation

### ISSUE THREAD

**Issue by Jeff09** [cardwing/Codes-for-PVKD#3]
Congrats! Is there any plan to release the code for Point-to-Voxel Knowledge Distillation except vanilla training code?

**Comment by cardwing** [comment]
The distillation codes will be released soon after the DDP codes and all pre-trained models are ready.

**Comment by Jeff09** [comment]
In the paper, the final loss function is composed of seven terms. Do you simultaneously train the student model and the teacher model or fix the teacher model only to train the student model?

**Comment by L-Reichardt** [comment]
@Jeff09 The model code here is the 2+ year old code from Cylinder3D, from the pre-CVPR Paper (missing pointwise refinement module). Without the proper model code the 2-part loss cannot be reconstructed.

The authors did give some instructions in 2021 on how this module could be implemented [ref. Issue](https://github.com/xinge008/Cylinder3D/issues/28), but unfortunately leave out some important things, such as how the shuffled data from PPmodel is handled.
